ui/components/icons.mjs

```diff
@@ -333,6 +333,16 @@ export const HardwareIcon = (props) => (
   </IconWrapper>
 )
 
+/*
+ * AddHardwareIcon - Looks like a chip
+ */
+export const AddHardwareIcon = (props) => (
```

Comment:
The `IconWrapper` will handle `props.children`, and if we add the same to `HardwareIcon` you should be able to do this:
```mjs
export const AddHardwareIcon = (props) => (
  <HardwareIcon {...props}>
    <path d="M22 6.25h-1.75V4.5h-1.5v1.75H17v1.5h1.75V9.5h1.5V7.75H22v-1.5Z" />
  </HardwareIcon>
)
```

Then in `HardwareIcon` add:
```
  {...props.children}
```
before the closing `</IconWrapper>` tag.
  
This keeps the code DRY